overview/intro.md

```diff
@@ -2,67 +2,113 @@
+### What is header bidding
 
-Sometimes when folks talk about our flagship "Prebid.js", they abbreviate it to "Prebid" for short, and this is fine, but we'd prefer that people remember we have other products too.
+Header bidding is a response to a fragmented and highly inefficient process for digital ad display. It is an alternative to the "waterfall" method, the process of offering impressions in one sales channel and if that does not succeed moving to less valuable channels. Header bidding is sometimes referred to as advance bidding or pre-bidding. 
+When the page loads, header bidding enables publishers to have simultaneous auctions with all SSPs and ad exchanges. Publishers can receive bids on their inventory that may be unavailable through their primary ad server and exchange.
+The returned bids are then passed into the ad server so they can compete with direct demand and the primary ad server's exchange on a level playing field.
 
+### A brief history of header bidding
 
-## What is Header Bidding?
+The early days of header bidding were dominated by bad practices, closed proprietary tech, poor standards, and little to no cooperation between competing companies. Publishers were presented with a confusing and time consuming process of having to manually patch together various solutions from different companies and processes.
+Prebid.js launched in 2015 to make header bidding easy for publishers by bringing conformity and simplictity to the header bidding process. By creating a simple, open tech layer upon which companies could add their code to a standard but optimized foundation, Prebid.js made it easier to implement header bidding, and offered the largest repository of working adapters.
+Today, Prebid.js is the most widely used header bidding "container" or "wrapper" on the web. The ecosystem supports more thabn 150 demand partners, over fifteen analytics providers, and numerous publishers.
+## Benefits of Prebid Products
 
-Header Bidding is a technique that involves running SSP & Ad Exchange code directly on page so publishers can receive bids on their inventory that may be unavailable through their primary ad server and exchange.
+The Prebid product suite offers publishers multiple benefits designed to foster a better header bidding experience:
+1. Free and open source, enabling anyone to contribute or review code.
+2. The largest repository of working header bidding adapters.
+3. Asynchronous and single time-out to provide a better user experience.
+4. Prebid Server to run faster auctions with more partners.
+5. Tools and analytics to optimize your setup.
+6. Multiple options on formats (display, video, native) and channels (mobile, web).
+7. A well established and helpful community which can advise on best practices and if needed provide professional setup and services.  
```

Comment:
this should be a bulleted list rather than a numbered list.

Reply:
Changed.